configs/DEMO_RISCV.yaml

```diff
@@ -26,11 +26,10 @@ Queue-Sizes:
   Load: 64
   Store: 36
 Branch-Predictor:
+  Type: "Perceptron"
   BTB-Tag-Bits: 11
-  Saturating-Count-Bits: 2
-  Global-History-Length: 10
-  RAS-entries: 5
-  Fallback-Static-Predictor: "Always-Taken"
+  Global-History-Length: 19
+  RAS-entries: 1
```

Comment:
I think this should be a point of discussion - if this predictor improves branch prediction rates, sometimes considerably, then why would we want to use a worse predictor?

Reply:
I guess whatever is most accurate to the model

Reply:
As long as it is closer to hardware results (not just better mispredict rates) then I'm happy for it to stay. Does @jj16791 have a table of hardware results for these codes?

Reply:
Sent offline

Reply:
Hardware results all below 5%; we are still well above in all cases. Happy to leave change as is. Once we do get to the similar misprediction rates, it may be important to use model accurate predictors to get the correct behaviour